desktop/platform/index.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+/**
+ * Internal dependencies
+ */
+const os = require( 'os' );
+
+/**
+ * Module variables
+ */
+let platform = false;
+
+function Platform() {
+	this.platform = false;
+}
+
+Platform.prototype.isOSX = function() {
+	return process.platform === 'darwin';
+};
+
+Platform.prototype.isWindows = function() {
+	return process.platform === 'win32';
+};
+
+Platform.prototype.isWindows10 = function() {
+	return parseInt( os.release(), 10 ) >= 10;
```

Comment:
I'll do something better. I'll remove the function. Not even calypso is using it :disappointed: